pages/linux/adig.md

```diff
@@ -11,14 +11,14 @@
 
 `adig -d {{example.com}}`
 
-- Connect to [s]pecified DNS server:
+- Connect to a specific DNS server:
 
 `adig -s {{1.2.3.4}} {{example.com}}`
 
-- Use specified TCP port to connect to DNS server:
+- Use a specific TCP port to connect to DNS server:
```

Comment:
```suggestion
- Use a specific TCP port to connect to the DNS server:
```

Reply:
I guess `a` is better here as more than one DNS server can be connected to.

Reply:
Yeah, that works too. Updated the suggestions.